appserver/tests/payara-samples/classpath/pom.xml

```diff
@@ -1,60 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>fish.payara.samples</groupId>
-        <artifactId>payara-samples</artifactId>
-        <version>5.201-SNAPSHOT</version>
-    </parent>
+	<parent>
+		<groupId>fish.payara.samples</groupId>
+		<artifactId>payara-samples</artifactId>
+		<version>5.201-SNAPSHOT</version>
+	</parent>
 
-    <artifactId>samples-classpath-checked</artifactId>
-    <packaging>pom</packaging>
-    <name>Payara Samples - Classpath Tests</name>
-    <description>
+	<artifactId>samples-classpath-checked</artifactId>
+	<packaging>pom</packaging>
+	<name>Payara Samples - Classpath Tests</name>
+	<description>
       These test verify that artifacts provide classpath isolated enough not to interfere
       with client code in unexpected ways.
     </description>
 
 	<profiles>
 		<profile>
-			<id>unstable</id>
-		    <modules>
-		        <module>embedded-vs-jersey</module>
-		    </modules>
+		<id>payara-server-embedded</id>
+			<modules>
+				<module>embedded-vs-jersey</module>
+			</modules>
 		</profile>
 	</profiles>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <inherited>false</inherited>
-                <executions>
-                    <execution>
-                        <id>enforce-clean-classpath</id>
-                        <goals>
-                            <goal>enforce</goal>
-                        </goals>
-                        <configuration>
-                            <rules>
-                                <bannedDependencies>
-                                    <excludes>
-                                        <exclude>*:*</exclude>
-                                    </excludes>
-                                    <includes>
-                                        <include>com.sun:tools-jar</include>
-                                        <include>junit:junit</include>
-                                        <include>org.hamcrest:hamcrest-core</include>
-                                    </includes>
-                                    <message>This module cannot inherit any dependencies to keep children's classpaths clean</message>
-                                </bannedDependencies>
-                            </rules>
-                            <fail>true</fail>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<inherited>false</inherited>
```

Comment:
Why is this being entirely converted to tabs?

Reply:
eclipse playing games